release/models/virtual-devices/openconfig-virtual-devices.yang

```diff
@@ -0,0 +1,386 @@
+// Copyright (c) 2022 Intel Corporation.
+// SPDX-License-Identifier: Apache-2.0
+
+module openconfig-virtual-devices {
+
+  yang-version "1";
+
+  // namespace
+  namespace "http://openconfig.net/yang/virtual-devices";
+
+  prefix "oc-vdevice";
+
+  // import some basic types
+  import openconfig-yang-types { prefix oc-yang; }
+  import openconfig-types { prefix oc-types; }
+  import openconfig-extensions { prefix oc-ext; }
+  import openconfig-virtual-device-types { prefix oc-vdevice-types; }
+
+  // meta
+  organization "IPDK - Infrastructure Programming Development Kit";
+
+  contact
+    "IPDK Admin
+    ipdkadmin@gmail.com";
+
+  description
+    "Model for managing virtual devices added/deleted
+    into compute instances such as VMs or whole hosts.";
+
+  oc-ext:openconfig-version "2.5.0";
+
+  // OpenConfig specific extensions for module metadata.
+  oc-ext:regexp-posix;
+  oc-ext:catalog-organization "openconfig";
+  oc-ext:origin "openconfig";
+
+  // typedef statements
+
+  typedef base-virtual-device-ref {
+    type leafref {
+      path "/oc-vdevice:virtual-devices/oc-vdevice:virtual-device/oc-vdevice:device-id";
+    }
+    description
+      "Reusable type for by-name reference to a base virtual device.";
+  }
+
+  typedef virtual-device-id {
+    type string;
+    description
+      "User-defined identifier for a virtual-device, generally used to
+      name a virtual device reference.  The id can be arbitrary but a
+      useful convention is to use a combination of device type
+      and numerical index.";
+  }
+
+  grouping vdevice-model-version {
+    description
+      "Reusable definition for a reference to a virtual device model version";
+
+      leaf model-version {
+        type string;
+        description
+          "Version number of config & oper objects";
```

Comment:
Maybe replace `&` with the word `and` and `open` with `operational`?

Reply:
Ack. Fixed.